opentelemetry-sdk/src/metrics/pipeline.rs

```diff
@@ -413,15 +412,18 @@ where
                 if existing == id {
                     return;
                 }
-
-                global::handle_error(MetricsError::Other(format!(
-                    "duplicate metric stream definitions, names: ({} and {}), descriptions: ({} and {}), kinds: ({:?} and {:?}), units: ({:?} and {:?}), and numbers: ({} and {})",
+                // If an existing instrument with the same name but different attributes is found,
+                // log a warning with details about the conflicting metric stream definitions.
+                otel_warn!(
+                    name: "Instrument.DuplicateMetricStreamDefinitions",
```

Comment:
not sure what is the action end users are expected to do when they see this.. If this is only for sdk maintainers to debug, then lets use debug level only. If keeping at Warn, then it should be actionable/descriptive.

Reply:
done